assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -44,7 +37,25 @@ class Footer extends React.Component {
                 bluenove
               </Link>
             </div>
-            {assemblVersion ? <div className="assembl-version">v{assemblVersion}</div> : null}
+            <div className="terms">
+              {termsOfUseUrl &&
+                <div className="terms-of-use">
+                  <Link to={termsOfUseUrl} target="_blank">
+                    <Translate value="footer.terms" />
+                  </Link>
+                </div>}
+              {termsOfUseUrl && legalNoticeUrl && <span style={{ padding: '0 5px' }}>-</span>}
```

Comment:
Can you please avoid inline style if possible?
For styling purpose, `&mdash;` may be better?

Reply:
Sure. Thanks for &mdash, didn't know about it